packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -562,6 +574,14 @@ export default ({ api }: { api: APIType }) => {
         })
       )
       yield put(A.fetchNftOrders())
+
+      const status: OpenSeaStatus = yield call(api.getOpenSeaStatus)
+      yield put(
+        A.setOpenSeaStatus({
+          // @ts-ignore
+          status
+        })
+      )
```

Comment:
Might not need this, just need to call `nftActions.fetchOpenSeaStatus()` from the component